### PR TITLE
[4.3] Media Manager error message

### PR DESCRIPTION
**Pull request by brianteeman**
This PR improves the message for the media manager when the file is too large by describing and adding a link to the media manager options

## To test
Go to the media manager component options and change the maximum size from the default of 10 to 1Mb Try to upload an image larger than 1MB

## Before
image

## After
image

### Link to documentations
Please select:
- [x] No documentation changes for docs.joomla.org needed

- [x] No documentation changes for manual.joomla.org needed

### CODE REVIEW

**Comment by Kostelano** [comment]
Language strings with such an implementation are poorly translated into other languages due to different declensions and, for example, it is also not customary to use capitalized words in the middle of a sentence. It's better to just use a variable as a reference.

Something like this:

`This file is too large to upload. You can change the limits for your site in the <a href=\"%s\">Media component</a>.`

Anticipating an objection, I will say that HTML is used all the time in language files.

**Comment by brianteeman** [comment]
It is a standard string construction used throughout Joomla. If you believe it is a bad structure for translators then please raise a new issue for that. It is beyond the scope of this PR.

**Comment by ChristineWk** [comment]
I have tested this item :white_check_mark: successfully on de90fddbeee2ce485e2b77a6992495a8872dbcfb<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39741">issues.joomla.org/tracker/joomla-cms/39741</a>.</sub>

File: administrator/components/com_media/src/Controller/ApiController.php [hunk 1]
```diff
@@ -349,14 +351,15 @@ private function checkContent()
         $uploadMaxFilesize   = $helper->toBytes(ini_get('upload_max_filesize'));
         $postMaxSize         = $helper->toBytes(ini_get('post_max_size'));
         $memoryLimit         = $helper->toBytes(ini_get('memory_limit'));
-
+        $link                = 'index.php?option=com_config&view=component&component=com_media';
+        $output              = HTMLHelper::_('link', Route::_($link), Text::_('JOPTIONS'));
```
Comment (Quy) [inline administrator/components/com_media/src/Controller/ApiController.php]: Move inside the if statement.

**Comment by toivo** [comment]
I have tested this item :white_check_mark: successfully on 9c1425ea6087b608a2c7a06a4e5eb7d798a63bf6<br /><br />Tested successfully in Joomla 4.3.0-beta2-dev of 16 February.<hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39741">issues.joomla.org/tracker/joomla-cms/39741</a>.</sub>

**Comment by toivo** [comment]
What happened to @ChristineWk's earlier successful test result? <hr /><sub>This comment was created with the <a href="https://github.com/joomla/jissues">J!Tracker Application</a> at <a href="https://issues.joomla.org/tracker/joomla-cms/39741">issues.joomla.org/tracker/joomla-cms/39741</a>.</sub>

**Comment by obuisard** [comment]
Thank you, Brian @brianteeman !

**Pull request merged**